BASE/src/Microsoft.ApplicationInsights/Metrics/Implementation/AutocollectedMetricsExtraction/ExceptionMetricsExtractor.cs

```diff
@@ -0,0 +1,153 @@
+
+                        IList<string> dimensionNames = new List<string>(this.dimensionExtractors.Count);
+                        for (i = 0; i < this.dimensionExtractors.Count; i++)
+                        {
+                            dimensionNames.Add(this.dimensionExtractors[i].Name);
+                        }
+
+                        MetricIdentifier metricIdentifier = new MetricIdentifier(MetricIdentifier.DefaultMetricNamespace,
+                                    MetricTerms.Autocollection.Metric.ExceptionCount.Name,
+                                    dimensionNames);
+
+                        this.exceptionServerMetric = metricTelemetryClient.GetMetric(
+                                                                    metricIdentifier: metricIdentifier,
+                                                                    metricConfiguration: config,
+                                                                    aggregationScope: MetricAggregationScope.TelemetryClient);
+                        this.isInitialized = true;
+                    }
+                }
+            }            
+        }
+
+        public void ExtractMetrics(ITelemetry fromItem, out bool isItemProcessed)
+        {
+            ExceptionTelemetry exception = fromItem as ExceptionTelemetry;
+            if (exception == null)
+            {
+                isItemProcessed = false;
+                return;
+            }
+
+            //// If there is no Metric, then this extractor has not been properly initialized yet:
+            if (this.exceptionServerMetric == null)
+            {
+                //// This should be caught and properly logged by the base class:
+                throw new InvalidOperationException(Invariant($"Cannot execute {nameof(this.ExtractMetrics)}.")
+                                                  + Invariant($" There is no {nameof(this.exceptionServerMetric)}.")
+                                                  + Invariant($" Either this metrics extractor has not been initialized, or it has been disposed."));
+            }
+
+            string[] dimValues = new string[this.dimensionExtractors.Count];
+            
+            for (int i = 0; i < this.dimensionExtractors.Count; i++)
+            {
+                var dim = this.dimensionExtractors[i];
+                if (dim.MaxValues == 0)
+                {
+                    dimValues[i] = MetricTerms.Autocollection.Common.PropertyValues.Other;
+                }
+                else
+                {
+                    dimValues[i] = dim.ExtractDimension(exception);
+                    if (string.IsNullOrEmpty(dimValues[i]))
+                    {
+                        dimValues[i] = dim.DefaultValue;
+                    }
+                }
+            }
+
+            CommonHelper.TrackValueHelper(this.exceptionServerMetric, exception.Duration.TotalMilliseconds, dimValues);
+            isItemProcessed = true;
```

Comment:
we extract just count of exceptions. No duration is involved here. (once you remove Exception from inherting OperationTelemetry, this would be addressed )

Reply:
Fixed